lib/editor/components/timetable/TimetableEditor.js

```diff
@@ -49,6 +49,7 @@ type Props = ContainerProps & {
   toggleAllRows: typeof tripActions.toggleAllRows,
   toggleDepartureTimes: typeof tripActions.toggleDepartureTimes,
   toggleRowSelection: typeof tripActions.toggleRowSelection,
+  toggleUseSeconds: typeof tripActions.toggleUseSeconds,
```

Comment:
It's only there for Prop typing, it's passed to the `TimetableHeader` through `{...this.props}` rather than explicitly. It's the same as currently exists for `toggleDepartureTimes`. Want that made explicit in the props for `TimetableHeader`?

Reply:
> Want that made explicit in the props for TimetableHeader?

That would be my preference. We don't need to prop drill the types.

Reply:
Updated in https://github.com/ibi-group/datatools-ui/pull/984/commits/48ff8db0265979ffe7e7de2f8fec111cc13f6b9d